query/stdlib/influxdata/influxdb/tag_values_measurement_or_predicate_test.flux

```diff
@@ -0,0 +1,125 @@
+package influxdb_test
```

Comment:
Eventually these tests should be ported to the `flux` repo, since it seems like it would make sense for them to be run across all platforms using flux.